ir_actions_report_xml_multi_company/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV, Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Ir Actions Report Xml Multi Company',
+    'summary': """
```

Comment:
This multi line string is introducing errant whitespace. Please use single line string, concatenated if multi lines are necessary:

```
'summary': 'my long'
           'summary',
```